ors-api/src/test/java/org/heigit/ors/apitests/isochrones/ResultTest.java

```diff
@@ -478,4 +489,90 @@ void testCompleteMetadata() {
                 .body("metadata.containsKey('system_message')", is(true))
                 .statusCode(200);
     }
+
+    @Disabled
```

Comment:
Why do we add a disabled test? Either remove the test or add a comment explaining what it is for.

Reply:
Good catch! It was primarily meant for debugging purposes, I've removed it.